ft-fine-tune-bert/fine-tune-unittests.py

```diff
@@ -1,55 +1,72 @@
+    #         ],
+    #      ]
 
-        with open("6_genre_clean_training_data_small_unittests.txt", "w") as output:
-            for row in self.train_data:
-                for col in row: 
-                    output.write(str(col) + '\n')
+    #     with open("6_genre_clean_training_data_small_unittests.txt", "w") as output:
+    #         for row in self.train_data:
+    #             for col in row: 
+    #                 output.write(str(col) + '\n')
 
-        with open("6_genre_eval_data_small_unittests.txt", "w") as output:
-            for row in self.eval_data:
-                for col in row:
-                    output.write(str(col) + '\n')
+    #     with open("6_genre_eval_data_small_unittests.txt", "w") as output:
```

Comment:
please remove/delete the commented code as its huge blocks and it confuses the person reading the code.